object-manipulation-2/src/pick.js

```diff
@@ -0,0 +1,23 @@
+/* exported pick */
+
+/*
+  - Create new object variable
+  - Loop through source object
+    - If first key is equal to current property,
+      - Then new object at that property is set to current key
+    - If second key is equal to current prop.
+      - Then new object at current prop is set to current key
+  - Return new object
+*/
+
+function pick(source, keys) {
+  var newObject = {};
+  for (var prop in source) {
+    if (keys[0] === prop) {
+      newObject[prop] = source[prop];
+    } else if (keys[1] === prop) {
+      newObject[prop] = source[prop];
```

Comment:
Same here. Try to think about the problem slightly differently. You might receive any number of keys.